pkg/timeout/handler.go

```diff
@@ -58,24 +58,37 @@ type Backoff struct {
 // backoff algorithm and returns the "jittered" result.
 type jitterFunc func(numSeconds int) (jitteredSeconds int)
 
-// TimeoutSet contains required k8s interfaces to handle build timeouts
-type TimeoutSet struct {
-	logger                  *zap.SugaredLogger
-	taskRunCallbackFunc     func(interface{})
+// Handler knows how to track channels that can be used to communicate with go routines
+// which timeout, and the functions to call when that happens.
+type Handler struct {
+	logger *zap.SugaredLogger
+
+	// taskRunCallbackFunc is the function to call when a TaskRun has timed out.
+	// This is usually set to the function that enqueues the taskRun for reconciling.
+	taskRunCallbackFunc func(interface{})
+	// pipelineRunCallbackFunc is the function to call when a TaskRun has timed out
+	// This is usually set to the function that enqueues the taskRun for reconciling.
```

Comment:
NIT: 
s/when a TaskRun/when a PipelineRun/
s/enqueues the taskRun/enqueues the pipelineRun/

Reply:
whoops, thanks @pritidesai !